pkg/util/roundtrippers/roundtrippers.go

```diff
@@ -0,0 +1,13 @@
+package roundtrippers
```

Comment:
maybe also move `retryingRoundTripper` from `pkg/cluster/kubeclient/roundtrip.go` into this file?